holoviews/plotting/bokeh/callbacks.py

```diff
@@ -92,19 +104,41 @@ class Callback(object):
     attributes = {}
 
     js_callback = """
+        function unique_events(events) {{
```

Comment:
Independent of the details of this function a set of JS utilities definitely makes sense. The issue then is that we will really want to have a proper JS package, because polluting the main namespace isn't ideal. All things to consider once we get started writing the widget manager.